tools/chocolateyInstall.ps1

```diff
@@ -1,7 +1,19 @@
-﻿$packageName = "postman"
+$packageName = "postman"
 $fileType = "exe"
 $silentArgs = "-s"
-$url = "https://dl.pstmn.io/download/latest/win32"
-$url64 = "https://dl.pstmn.io/download/latest/win64"
+$url = "https://dl.pstmn.io/download/version/6.5.2/windows32"
+$url64 = "https://dl.pstmn.io/download/version/6.5.2/windows64"
```

Comment:
This package is intended to install the latest version available, even if the package itself is out of date. This reduces potential user frustration with installing an outdated version, only to have an update immediately available.

This strategy could work if the package was automatically updated, but I don't currently have anything like that setup. Ultimately, my main focus is delivering the latest.

Thanks for letting me know about the newer version though, https://chocolatey.org/packages/postman/6.5.2 is published.